### PR TITLE
Added a note to use a legacy CLI version for WCF projects

### PR DESCRIPTION
**Pull request by OliaG**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/porting/upgrade-assistant-install.md](https://github.com/dotnet/docs/blob/968a5f200e2fbe25c58f7953a7371fa6f049c4ef/docs/core/porting/upgrade-assistant-install.md) | [Install the .NET Upgrade Assistant](https://review.learn.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-install?branch=pr-en-us-35712) |

### CODE REVIEW

File: docs/core/porting/upgrade-assistant-install.md [hunk 1]
```diff
@@ -51,3 +51,5 @@ dotnet tool update -g upgrade-assistant
 > ```dotnetcli
 > dotnet tool install -g --ignore-failed-sources upgrade-assistant
 > ```
+
+For upgrading projects with WCF components, use the older version of the CLI tool. See (Install (legacy version))[https://learn.microsoft.com/dotnet/core/porting/upgrade-assistant-install-legacy] for more details.
```
Comment (OliaG) [inline docs/core/porting/upgrade-assistant-install.md:55]:
```suggestion
For upgrading projects with WCF components, use the older version of the CLI tool. See [Install (legacy version])(https://learn.microsoft.com/dotnet/core/porting/upgrade-assistant-install-legacy) for more details.
```

**Review by OliaG — COMMENTED** [review]
LGTM

File: docs/core/porting/upgrade-assistant-install.md [hunk 2]
```diff
@@ -51,3 +51,5 @@ dotnet tool update -g upgrade-assistant
 > ```dotnetcli
 > dotnet tool install -g --ignore-failed-sources upgrade-assistant
 > ```
+
+For upgrading projects with WCF components, use the older version of the CLI tool. See [Install (legacy version])(https://learn.microsoft.com/dotnet/core/porting/upgrade-assistant-install-legacy) for more details.
```
Comment (gewarren) [inline docs/core/porting/upgrade-assistant-install.md:55]:
```suggestion
For upgrading projects with WCF components, use the older version of the CLI tool. For more information, see [Install (legacy version)](upgrade-assistant-install-legacy.md).
```

**Comment by mconnew** [comment]
@gewarren, the preview link generated in the initial comment hasn't updated with the correction of how the link was created. Is there a way to make it regenerate?

**Comment by gewarren** [comment]
> @gewarren, the preview link generated in the initial comment hasn't updated with the correction of how the link was created. Is there a way to make it regenerate?

It looks like it's been updated.
https://review.learn.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-install?branch=pr-en-us-35712#install-the-net-global-tool

**Review by mconnew — APPROVED** [review]

**Pull request merged**